### ISSUE TITLE
Combobox not rendered correctly (on Chrome and Jupyter Notebook)

### ISSUE THREAD

**Issue by tkrabel-db** [jupyter-widgets/ipywidgets#3359]
## Description

When displaying a Combobox on Jupyter Notebook, there is a dropdown arrow displayed which is misaligned.

<img width="986" alt="Screenshot 2022-01-26 at 17 39 12" src="https://user-images.githubusercontent.com/91616041/151336782-702f4318-da33-4df2-8f81-340790c72379.png">

BTW: You see the same misalignment on the [ipywidgets docs page](https://ipywidgets.readthedocs.io/en/stable/examples/Widget%20List.html#Combobox)

<img width="754" alt="Screenshot 2022-01-27 at 11 04 53" src="https://user-images.githubusercontent.com/91616041/151337143-89b75edf-811b-4dc4-b650-ef2e60a613ee.png">

## Reproduce

1. Open Jupyter Notebook on Chrome
2. Type
```python
widgets.Combobox(
    # value='John',
    placeholder='Choose Someone',
    options=['Paul', 'John', 'George', 'Ringo'],
    description='Combobox:',
    ensure_option=True,
    disabled=False
)
```

## Context

- ipywidgets version 7.6.5
- Jupyter Notebook version: 6.4.5
- Operating System and version: MacOS bigSur 11.6.2
- Browser and version: Google Chrome 97.0.4692.99

**Comment by jasongrout** [comment]
I looked into this briefly. As background, the combobox is implemented using the browser [datalist](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist) element.

First, this is not a problem on Firefox (which does not have an arrow) and not a problem on Safari (which correctly has a vertically centered arrow)

Looking into the Chrome browser debugger, it seems that if we reset the line-height (to `initial`) for the combobox `input` element, the down arrow is centered. Currently the line-height is set for this widget at https://github.com/jupyter-widgets/ipywidgets/blob/1b1a49d39efb24d24422516d2730c73dfdec9d84/packages/controls/css/widgets-base.css#L614
As far as I can tell, deleting that line-height CSS does not change anything since the height is also specified.

Marking as a good first issue.

**Comment by jasongrout** [comment]
Fixed in #3367
